templates/careers/support-engineering.html

```diff
@@ -1,88 +1,28 @@
-      </div>
-      <div class="col-6">
-        <p>In Support Engineering, you will enjoy the excitement of a team that is fun, diverse, distributed globally and incredibly competent. Every day, we deliver the expertise our customers need to solve some of the world’s most challenging problems.</p>
-        <p>While we all enjoy the feeling that comes from passionately working together, we also collectively respect each other's lives away from work. Part of standing together with a global team that is set up to work remotely means you can step away during your off-hours and know that your team has your back. Your manager will guide and mentor you through your work but equally wants you to make sure you are taking the time away from work to help you unwind and refocus. All of us are in this together, and we want you in Support Engineering for the long term.</p>
-      </div>
-    </div>
-  </div>
-  <div class="p-strip is-shallow u-no-padding--top">
-    <div class="u-fixed-width u-hide--small u-hide--medium">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/456c9c0f-department-support-engineering-%402x.jpg",
-        alt="",
-        width="1242",
-        height="467",
-        hi_def=True,
-        loading="auto"
-        ) | safe
-      }}
-    </div>
-  </div>
-  <hr class="is-fixed-width u-no-margin--bottom">
-  <div class="p-strip is-shallow u-no-padding--top">
-    <div class="row">
-      <div class="col-6">
-        <h3 class="p-heading--2">Projects you'll work on</h3>
-      </div>
-      <div class="col-6">
-        <p>We build strong relationships with customers and stakeholders. This may involve regular check-ins with key customers, hosting customer events, or performing service reviews.</p>
-        <p>We work closely with engineering teams to ensure that customer issues are addressed in a timely manner, and that support requirements are considered during the product development process.</p>
-        <p>Our team also improves the quality of support documentation and resources. This may involve updating existing documentation, creating new knowledge base articles, or developing tutorials and training materials.</p>
-      </div>
-    </div>
-  </div>
-  <div class="p-strip is-shallow u-no-padding--top">
-    <hr class="is-fixed-width u-no-margin--bottom">
-    <div class="row">
-      <div class="col-6">
-        <h3 class="p-heading--2">Who you are</h3>
-      </div>
-      <div class="col-6">
-        <p>If you enjoy helping customers, solving really complex problems and thrive on operating software or building software, then Support Engineering at Canonical offers a challenging work environment where you can learn and grow.</p>
-      </div>
-    </div>
-  </div>
-</section>
+{% block meta_description %}Canonical's Engagement Project Management Office team owns, drives and communicates delivery excellence by providing strong Project process guidelines, project continuity and data analytics.{% endblock %}
```

Comment:
```suggestion

```
This page doesn't have a description unfortunately. Just left [another comment](https://docs.google.com/document/d/1TkePZPzjdIMRTPIep6C1f8SnTPmkKiGdMyUEPMK9rH8/edit?disco=AAAAiXFCCqk) asking for it